shin-core/src/format/scenario/info.rs

```diff
@@ -80,34 +115,220 @@ impl MovieInfoItem {
+}
+
+impl FinalSegment for CharsSpriteSegment {
+    fn is_final(&self) -> bool {
+        match self {
+            CharsSpriteSegment::End => true,
+            _ => false,
+        }
+    }
 }
-pub type Section64Info = Vec<Section64InfoItem>;
 
+/// The data for a character in the Characters screen (`chars`)
 #[derive(Debug, BinRead, BinWrite)]
-pub struct Section68InfoItem {
-    pub unk1: u16,
-    pub unk2: u16,
-    pub unk3: u16,
+pub struct CharsSpriteInfoItem {
+    /// The episode for which the character sprite and description is valid.
+    pub episode: u8,
+
+    /// The segments defining the sprites and description for this character.
+    #[br(parse_with = parse_terminated_segment_list)]
+    pub segments: Vec<CharsSpriteSegment>,
 }
-pub type Section68Info = Vec<Section68InfoItem>;
 
+pub type CharsSpriteInfo = Vec<CharsSpriteInfoItem>;
+
+/// An individual instruction for building the data underlying the grid in the Characters screen (`chars`).
+#[derive(Debug, BinRead, BinWrite)]
+pub enum CharsGridSegment {
+    /// Ends the current character grid definition.
+    #[brw(magic = 0x0u8)]
+    End,
+
+    /// Defines a portrait on the grid, with a corresponding character to have its full sprite, name, and description shown when the portrait is selected.
+    #[brw(magic = 0x1u8)]
+    Portrait {
+        /// The page to show this portrait on, between 0 and 3 (both inclusive). The game will automatically generate as many pages as necessary for a specific grid.
+        page: u8,
+
+        /// The portrait's X position on the grid.
+        grid_x: u8,
+
+        /// The portrait's Y position on the grid.
+        grid_y: u8,
+
+        /// The ID of the character this portrait is for, indexing into [`CharsSpriteInfo`].
+        character_id: u16,
+
+        /// Defines how the portrait and the corresponding sprite is displayed, and how it behaves on Execute/Resurrect:
+        ///  - **in game**: `0` = empty portrait, `1` = alive, `2` = dead (small blood splotch on sprite), `3` = very dead (large blood splotch on sprite)
+        ///  - **out of game** (Characters screen selected from main menu): `0` = empty portrait, `1` = two phases (alive/dead), `2` = three phases (alive/missing/dead)
+        behaviour: u8,
+
+        /// Modifies the `behaviour` in certain circumstances.
+        behaviour_modifier: u8, // todo: find out and document what this does
+    },
+
+    /// Defines an individual line segment to be placed on the grid, to connect character portraits.
+    #[brw(magic = 0x2u8)]
+    Connector {
+        /// The page to show this portrait on, between 0 and 3 (both inclusive). The game will automatically generate as many pages as necessary for a specific grid.
+        page: u8,
+
+        /// The connector's X position on the grid.
+        grid_x: u8,
+
+        /// The connector's Y position on the grid.
+        grid_y: u8,
+
+        /// The shape of this connector:
```

Comment:
Symbols from https://www.compart.com/en/unicode/block/U+2500 might make the docs less ambiguous